pkgs/by-name/od/odin/package.nix

```diff
@@ -24,17 +21,6 @@ stdenv.mkDerivation {
     hash = "sha256-GXea4+OIFyAhTqmDh2q+ewTUqI92ikOsa2s83UH2r58=";
   };
 
-  postPatch =
-    lib.optionalString stdenv.hostPlatform.isDarwin ''
-      substituteInPlace src/linker.cpp \
-          --replace-fail '/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk' ${MacOSX-SDK}
-    ''
```

Comment:
Here’s the relevant piece of code: https://github.com/odin-lang/Odin/blob/4374685007fbf0398edfc8c94c39a8470572d9d0/src/linker.cpp#L772-L782.

This patching does currently have an effect; it replaces the `/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk` reference (problematic for building packages in Nix from a purity perspective) with an inert stub. However, the mentions of `/usr/local`, `/opt/homebrew`, and `/opt/local` seem comparably problematic. I suppose it is okay to leave the former unpatched since the latter already create impurities and the only Odin package we have in tree is `ols`, but I would personally be tempted to patch those lines out entirely.

Reply:
Correct. Updating.